docs/docs/using-cloudinary-image-service.md

```diff
@@ -0,0 +1,141 @@
+
+## Solutions Cloudinary provides 
+
+Cloudinary provides a couple of amazing solutions to this problem , namely: 
+- Remote storage and delivery of images via CDN
+- Wider range of transformations over [gatsby-image](/docs/using-gatsby-image/).
+- [Digital Asset Management](https://cloudinary.com/documentation/digital_asset_management_overview) for enterprise assets 
+
+## Gatsby-source-cloudinary
+
+This plugin fetches media assets from Cloudinary that are specified in a folder. It then transforms these images into Cloudinary file nodes, which can be queried with GraphQL in a Gatsby project. 
+`gatsby-source-cloudinary` applies [f_auto and q_auto](https://cloudinary.com/documentation/image_transformations) transformation parameters which aid in automatic optimisation of format and quality for media assets by over 70 percent.
+
+### Prerequisites  
+
+Before using the `gatsby-source-cloudinary` you should do the following : 
+
+- Upload the images to a folder on Cloudinary. This folder can have any name of your choosing. 
+- Obtain your API key and API secret from your Cloudinary dashboard.
+- Have [dotenv](https://www.npmjs.com/package/dotenv) module installed for loading environment variables from a `.env` file.
+
+### Using gatsby-source-cloudinary
+
+Add the plugin to your project in these steps : 
+
+1. Install `gatsby-source-cloudinary` 
+
+```shell
+  npm install gatsby-source-cloudinary
+```
+2. In the root of your project, create an environment file called `.env` and add your Cloudinary credentials and their values
+
+```
+CLOUDINARY_API_KEY=xxxxxxxxxxxxxx
+CLOUDINARY_API_SECRET=xxxxxxxxxxxxxxxxxxxx
+CLOUDINARY_CLOUD_NAME=xxxxx
+```
+
+3. Configure `gatsby-config.js`
+
+```js:title=gatsby-config.js
+require('dotenv').config(); // highlight-line
+module.exports = {
+  ...
+  plugins:[
+  ...
+  {
+      resolve: `gatsby-source-cloudinary`,
+      options: {
+        cloudName: process.env.CLOUDINARY_CLOUD_NAME,
+        apiKey: process.env.CLOUDINARY_API_KEY,
+        apiSecret: process.env.CLOUDINARY_API_SECRET,
+        resourceType: `image`,
+        prefix: `gatsby-source-cloudinary/` 
+      }
+    }
+  ]
+} 
+```
+> `gatsby-source-cloudinary` takes these options:
+
+- **`cloudName`** , **`apiKey`** , and **`apiSecret`** **:** These are credentials from your Cloudinary console, stored as three separate environment variables for security.
+- **`resourceType`** **:** This is the resource type of the media assets: either an image or a video.
+- **`prefix`** **:** This is the folder (in your Cloudinary account) in which the files reside. In the example above, I named this folder `gatsby-source-cloudinary` . Assign a name of your choice.
+Other optional options are `type`, `tags`, and `maxResult`.
+
+> The `info` log, which displays the Cloudinary Media nodes. Those images are ready to be queried in Gatsby components.
```

Comment:
Not sure what the context for this statement is. Is this another option?

Reply:
I can take it out